### PR TITLE
establish fsync interfaces and general usage

### PR DESCRIPTION
**Pull request by daniel-garcia**
see Readme
https://github.com/infobloxopen/hotload/blob/fsync_stubs/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,68 @@
...
+    "database/sql"
+
+    log "github.com/sirupsen/logrus"
+    "github.com/infobloxopen/hotload"
+    _ "github.com/infobloxopen/hotload/fsnotify"
+    "github.com/lib/pq"
+)
+
+func init() {
+    hotdriver.Register("postgres", pq.Driver{})
```
Comment (kd7lxl) [inline README.md:21]: It's not obvious to me which import provides the `hotdriver` package.

**Review by kd7lxl — APPROVED** [review]
The interface LGTM.

File: README.md (hunk 1 above)
Comment (daniel-garcia) [reply README.md:21]: I'll add some comments

Comment (daniel-garcia) [reply README.md:29]: address in https://github.com/infobloxopen/hotload/pull/1/commits/67057caf6291c218f0487854c02ff9cd66437072

Comment (kd7lxl) [reply README.md:29]: The comments didn't help me, since they still don't mention `hotdriver`. Can you add a reference of where to find `hotdriver`?

Comment (daniel-garcia) [reply README.md:29]: I fixed the references. I had renamed the project from hotdriver to hotload.

**Pull request merged**